Header header

#Массив значений статусов ключей системы, 0 - выкл. 1 - вкл.
bool[16] status